Application/EdFi.Ods.SandboxAdmin/Options.cs

```diff
@@ -0,0 +1,26 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using CommandLine;
+
+namespace EdFi.Ods.SandboxAdmin
+{
+    public sealed class Options
+    {
+        [Option(
+            "exitAfterSandboxCreation",
+            Default = false,
+            HelpText = "exit the application after sandbox creation is complete (default: false)"
+        )]
+        public bool ExitAfterSandboxCreation { get; set; }
+        
+        [Option(
+            'e',
+            "environment",
+            Default = "production",
+            HelpText = "the execution environment (default: production)")]
+        public bool CommandLineEnvironment { get; set; }

```

Comment:
I could not find where this is used. Should it be removed?

Reply:
There is a line here that checks for "development" in the CLI arguments https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/943/files#diff-a6be1ebab450ff771bf0d245bddc35c1e36aaa89e26fab830df07237edfe85faL23 . I added this to include it in the help text since the argument is used before parsing occurs.

Reply:
Could the args be parsed before configuring the logging?

Reply:
Theoretically, it could prevent the settings in log4net.development.config (rather than log4net.config) from being applied to logging a problem encountered when parsing the other command line arguments. I agree that it would be better to use the parsed arguments, and that downside doesn't outweigh parsing consistently, so I changed it.